NOTES-NONSTOP.md

```diff
@@ -0,0 +1,148 @@
+ this document can cause application instability. The recommended approach
+ is to specify the OpenSSL version and threading model in your configuration
+ options, and keeping your memory and float options consistent, for example:
+ 
+ * For 1.1 `--prefix=/usr/local-ssl1.1 --openssldir=/usr/local-ssl1.1/ssl` 
+ * For 1.1 PUT `--prefix=/usr/local-ssl1.1_put --openssldir=/usr/local-ssl1.1_put/ssl`
+ * For 3.0 `--prefix=/usr/local-ssl3.0 --openssldir=/usr/local-ssl3.0/ssl` 
+ * For 3.0 PUT `--prefix=/usr/local-ssl3.0_put --openssldir=/usr/local-ssl3.0_put/ssl`
+
+ Use the `_RLD_LIB_PATH` environment variable in OSS to select the appropriate
+ directory containing `libcrypto.so` and `libssl.so`. In GUARDIAN, use the
+ `=_RLD_LIB_PATH` search define to locate the GUARDIAN subvolume where OpenSSL
+ is installed.
+
+Float Considerations
+--------------------
+
+ OpenSSL is built using IEEE float mode by default. If you need a different
+ IEEE mode, create a new configuration specifying `tfloat-x86-64` (for Tandem
+ float) or `nfloat-x86-64` (for Neutral Float).
+ 
+Memory Models
+-------------
+
+ The current OpenSSL default memory model uses the default platform address
+ model. If you need a different address model, you must specify the appropriate
+ c99 options for compile (`CFLAGS`) and linkers (`LDFLAGS`).
+
+Cross Compiling with NSDEE
+--------------------------
+
+ **Note:** None of these builds have been tested by the platform maintainer and are
+ supplied for historical value. Please submit a Pull Request to OpenSSL should
+ these need to be adjusted.
+ 
+ If you are attempting to build OpenSSL with NSDEE, you will need to specify
+ the following variables. The following set of compiler defines are required:
+ 
+`   COMP_ROOT=$(cygpath -w /path/to/comp_rooot)   # must be path format for system (ie windows)`
+`   CC=/path/to/c99                               # must be executable by shell`
+ 
+### Optional Build Variables 
+
+`   DBGFLAG="--debug"`
+`   CIPHENABLES="enable-ssl3 enable-ssl3-method enable-weak-ssl-ciphers enable-rc4"`
+
+### Internal Known On Platform Cross Compile Variables
+
+  The following definition is required if you are building on TNS/X for TNS/E
+  and have access to a TNS/E machine on your EXPAND network - with an example
+  node named \CS3:
+   
+`   SYSTEMLIBS="-L/E/cs3/usr/local/lib"`
+
+Version Procedure (VPROC) Considerations
+----------------------------------------
+
+  If you require a VPROC entry for platform version identification, use the
+  following variables:
+
+### For Itanium
+`    OPENSSL_VPROC_PREFIX=T0085H06`
+### For x86
+`    OPENSSL_VPROC_PREFIX=T0085L01`
+
+### Common Definition
+```
+    export OPENSSL_VPROC=${OPENSSL_VPROC_PREFIX}_$(cat include/openssl/opensslv.h |\
+      sed -n -e 's/^ *# *define  *OPENSSL_VERSION_TEXT[^"]*"\([^"]*\)"/\1/p' |\
+      sed -e 's/[. ]/_/g' -e 's/[.-]/_/g' |\
+      grep -v fips \
+      )
+```
```

Comment:
Since you have indented the shell lines with 4 spaces already, you probably don't need the tripple backquotes

Reply:
@rsbeckerca we try to use "minimalistic markdown" to optimize for reading both versions of the file, the raw ASCII version, and the rendered HTML. For more details, please read the commit messages of  

- 5f8e6c50bd862d6bb0bbd16b4c0b3be841d9cf32 and
- 6c8149df1fb6fce50a914a70040955d3512b0bd6 (in the given order).

In particular, as @levitte already indicated, we prefer to avoid triple backquotes, (` ``` `, a.k.a code fences) by indenting four characters.  (This extra indentation will be taken away; if you wan't to indent the rendered text by four characters, use eight characters. But it's not necessary IMHO)